ibm/mas_devops/roles/ocp_provision/templates/ipi/install-config.yaml.j2

```diff
@@ -31,3 +31,7 @@ platform:
 {% endif %}
 publish: External
 pullSecret: '{{ lookup('file', ipi_pull_secret_file) | regex_replace('\'', '"') }}'
+{% if sshKey != "" %}
+sshKey: |
+  {{sshKey}}
+{% endif %}
```

Comment:
Linting ... You've lost the empty newline at the bottom here as part of this update.